src/components/NewMessageDetail.vue

```diff
@@ -145,7 +145,7 @@ export default {
 				return
 			}
 			logger.debug('the  draft ID changed, we have to fetch the draft', {
-				currentId: this.draft.databaseId,
+				currentId: (this.draft !== undefined) ? this.draft.databaseId : 'no draft currently loaded',
```

Comment:
```suggestion
				currentId: this?.draft?.databaseId: 'no draft currently loaded',
```
if you don't mind :)

Reply:
you're right: it's nicer this way.

PR updated